RFS/src/main/java/com/rfs/RunRfsWorker.java

```diff
@@ -91,6 +97,14 @@ public static class Args {
 
         @Parameter(names = {"--log-level"}, description = "What log level you want.  Default: 'info'", required = false, converter = Logging.ArgsConverter.class)
         public Level logLevel = Level.INFO;
+
+
+        @Parameter(required = false,
```

Comment:
Blocking; I know how to visualize the output of these instrumentation, can you update the readme or update the code to allow for a very basic log things to files?  I am OK if the dump isn't a file but a running Jager instance as long as its easy for me to spin up and connect.

Reply:
Resolved by this PR https://github.com/gregschohn/opensearch-migrations/pull/4

Reply:
Since I was able to create that PR I do not consider this to be a blocking issue anymore - resolving this thread.